packages/eds-color-palette-generator/src/components/ColorManagement.tsx

```diff
@@ -46,15 +46,18 @@ export const ColorManagement = ({
                 />
               </div>
 
-              {/* Color Picker */}
+              {/* Color Hex Input */}
```

Comment:
Can we input colours in OKLCH as well? Since we’re promoting the use of OKLCH, can we use that as default, with the additional option of using HEX values?

Reply:
Created this: https://github.com/equinor/design-system/issues/4024